lib/block/blockupload.js

```diff
@@ -178,7 +174,7 @@ class BlockUpload extends EventEmitter {
             const pipeline = new Pipeline(
                 new FailUnsupportedAssets(),
                 new CreateTransferParts({ preferredPartSize }),
-                new MapConcurrent(new Transfer(randomFileAccess, retryOptions), { maxConcurrent }),
+                new MapConcurrent(new Transfer(randomFileAccess, options), { maxConcurrent }),
```

Comment:
this also inlcudes things like `method` since we supported other methods like `POST` instead of `PUT` before